src/main/resources/org/sagebionetworks/web/client/widget/entity/VersionHistoryWidgetViewImpl.ui.xml

```diff
@@ -5,45 +5,51 @@
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
```

Comment:
Ugly diff. Changes:
* Wrapped content in Collapse > FlowPanel, to match the other collapsible widgets
* Replaced class 'highlight-box' with 'light-border' to preserve appearance and position the close button
* Added a spot for the title and close button, since we're no longer using the highlight-box functionality to insert a title